references/segmentation/train.py

```diff
@@ -36,7 +36,8 @@ def get_transform(train, args):
         weights = torchvision.models.get_weight(args.weights)
         trans = weights.transforms()
 
-        def preprocessing(img, target):
+        def preprocessing(sample):
+            img, target = sample
```

Comment:
I didn't want to touch this any more than this, because that would mean I would need to mess with the presets transform on the new model weights. IIUC this is only for testing against pre-trained weights and thus out of scope of this preliminary test anyways.

Reply:
Do you then mean that this wont work any more? If it works, I'm fine with it.